src/Sylius/Bundle/ResourceBundle/composer.json

```diff
@@ -38,6 +39,7 @@
     "require-dev": {
         "phpspec/phpspec": "^2.4",
         "akeneo/phpspec-skip-example-extension": "~1.2",
+        "mongofill/mongofill": "dev-master",
```

Comment:
i know i recommended this in the other issue, but what is it for? the tests pass without this or the mongo extention